pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -279,6 +293,57 @@ spec:
             {{- end }}
             {{- end }}
 
+  - name: AWSCT_CNIIngressRules_Calico
+    enabledIf: '{{if (eq .cni "calico") }}true{{end}}'
+    definitions:
+    - selector:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: AWSClusterTemplate
+        matchResources:
+          infrastructureCluster: true
+      jsonPatches:
+      - op: add
+        path: "/spec/template/spec/network/cni/cniIngressRules/-"
```

Comment:
My style guide recommends not using `"` when not necessary, this should just be `path: /spec/template/spec/network/cni/cniIngressRules/-`